zfit/models/physics.py

```diff
@@ -548,3 +548,299 @@ class GeneralizedCBPDFRepr(BasePDFRepr):
 
 
 GeneralizedCB.register_analytic_integral(func=generalized_crystalball_mu_integral, limits=crystalball_integral_limits)
+
+
+@z.function(wraps="tensor", keepalive=True)
+def gaussexptail_func(x, mu, sigma, alpha):
+    t = (x - mu) / sigma * tf.sign(alpha)
+    abs_alpha = znp.abs(alpha)
+    cond = tf.less(t, -abs_alpha)
+    func = z.safe_where(
+        cond,
+        lambda t: znp.exp(0.5 * znp.square(abs_alpha) + abs_alpha * t),
+        lambda t: znp.exp(-0.5 * znp.square(t)),
+        values=t,
+        value_safer=lambda t: znp.ones_like(t) * abs_alpha,
+    )
+    return znp.maximum(func, znp.zeros_like(func))
```

Comment:
two refactoring ideas:
- we can move the znp.exp outside, after the ` safe_where`?
- why do we need the znp.maximum? exp(something) is always positive?

Reply:
I stole it from the crystal ball implementation. I can remove it